server.js

```diff
@@ -14,6 +14,10 @@ app.listen(port)
 
 const versoSpoof = require('./src/versoSpoof.js')
 
+app.get('/api/search', (req, res) => {
```

Comment:
No, this doesn't implement the real ES proxy yet. This PR is to demonstrate a pattern that I will iterate on once #630 is unblocked. (Small PRs good, right?)